server/models.py

```diff
@@ -6,11 +6,19 @@
 
 class Lecturer(db.Model):
     id = db.Column(db.Integer, primary_key=True)
+
+    user_name = db.Column(db.String(200), unique=True, nullable=False)
+    password = db.Column(db.String(200), nullable=False)
```

Comment:
Yes, more right way to do this, I think is to create `User` model with roles as `lecturer` and `student`. But for now we don't have students in our system and it was easier to do like this. We can change it in future, I think.